test/cqlpy/test_ssl.py

```diff
@@ -92,7 +93,7 @@ def test_system_clients_stores_tls_info(cql):
                 ):
                     return
                 time.sleep(0.01)
-            pytest.fail(f"Not all connections have TLS data set correctly in system.clients after 10s seconds")
+            pytest.fail(f"Not all connections have TLS data set correctly in system.clients after 10 seconds")
```

Comment:
Please note that this commit message only explains one of several changes, it's really odd (but it's easy to understand anyway, so I won't insist).